lgc/builder/YCbCrAddressHandler.cpp

```diff
@@ -47,19 +47,63 @@ void YCbCrAddressHandler::genBaseAddress(unsigned planeCount) {
   // For YCbCr, the possible plane counts are among 1 and 3.
   assert(planeCount > 0 && planeCount < 4);
 
-  // PlaneBaseAddresses[0] is the same as original base addressed as passed in SRD
-  m_planeBaseAddresses.push_back(m_regHandler->getReg(SqRsrcRegs::BaseAddress));
+  // PlaneBaseAddresses[0] is the same as original base addressed as passed in SRD, and is already pipeBankXored
+  Value *virtualAddrPlane0 = m_regHandler->getReg(SqRsrcRegs::BaseAddress);
+  m_planeBaseAddresses.push_back(virtualAddrPlane0);
 
+  Value *pipeBankXor1 = nullptr;
+  Value *pipeBankXor2 = nullptr;
+  Value *pipeBankXorNone = m_builder->getInt32(0);
+
+  switch (m_gfxIp->major) {
+  case 9: {
+    pipeBankXor1 = pipeBankXorNone;
+    pipeBankXor2 = pipeBankXorNone;
+    break;
+  }
+  case 10: {
+    // Judge if current swizzle mode is ADDR_SW_64KB_R_X
+    Value *isSW_64KB_R_X_Mode = m_builder->CreateICmpEQ(m_swizzleMode, m_builder->getInt32(27));
```

Comment:
Is there some enum or a constant you can use instead of `27`? If not, can you reference some docs that mention that `ADDR_SW_64KB_R_X` maps to `27`?

Reply:
Yes, there's exit enum in lgc/imported/chip/gfx9/gfx9_plus_merged_enum.h, I'll use that here.